hazelcast/src/test/java/com/hazelcast/map/LocalMapStatsTest.java

```diff
@@ -47,21 +50,48 @@
 
     HazelcastInstance instance;
     private String mapName = "mapName";
+    private String mapWithObjectFormat = "mapWithObjectFormat";
 
     @Before
     public void setUp() {
-        instance = createHazelcastInstance(getConfig());
+        instance = createHazelcastInstance(createMemberConfig());
     }
 
-    protected LocalMapStats getMapStats() {
+    protected final LocalMapStats getMapStats() {
+        return getMapStats(mapName);
+    }
+
+    protected LocalMapStats getMapStats(String mapName) {
```

Comment:
Hi @erosb,
These changes also need to be addressed on the enterprise side since there are some enterprise tests that extends this class.

Reply:
Hello @ufukyilmaz @mmedenjak here is the compilation fix PR, please review: https://github.com/hazelcast/hazelcast-enterprise/pull/4125